test/system/screenshots_test.rb

```diff
@@ -160,5 +160,15 @@ class ScreenshotsTest < ApplicationSystemTestCase
     visit node.path
     take_screenshot
   end
-  
+
+  test 'simple editor post' do
+    visit '/'
+    click_on 'Login'
+    fill_in("username-login", with: "steff1")
+    fill_in("password-signup", with: "secretive")
+    click_on "Log in"
+    visit(simple_editor_path)
```

Comment:
Aha what if you say `visit('/post/simple')` here, and don't use the Rails path generator?

Reply:
Having the path as 'post/simple` is causing the error...It is also visible on my server locally..I didn't catch this earlier :disappointed_relieved:  I think its a rails error... :thinking: because we do not have a post resource...it expects this to be like `editor/simple` because we have editor controller...maybe